ports/atmel-samd/boards/robohatmm1/mpconfigboard.mk

```diff
@@ -0,0 +1,34 @@
+LD_FILE = boards/samd21x18-bootloader-external-flash-crystalless.ld
+USB_VID = 0x0005
```

Comment:
Do you actually own this VID?

Reply:
No.  This is a placeholder until we get a new VID/PID for each product.  What should it be set to if we do not have a VID yet?

I'm hoping to have a real one before the end of next week.

Reply:
We have just put in a request for VID: 0x4D41 with a swag of PIDs.  I will update once confirmed.